app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class NotesController < ApplicationController
+  def create
+    volunteer = Volunteer.find(params[:volunteer_id])
+    volunteer.notes.create(note_params)
```

Comment:
This could error with an invalid volunteer ID but I don't expect that to happen so it's fine as is.